config/deploy.rb

```diff
@@ -32,9 +32,27 @@
 
 set :keep_releases, 5
 
+set :npm_target_path, -> { release_path.join('frontend') } # default not set
+set :npm_flags, ''
+
 namespace :deploy do
   after :finishing, 'deploy:cleanup'
   after 'deploy:publishing', 'deploy:restart'
 end
 
-set :rvm_ruby_version, '2.4.1'
\ No newline at end of file
+namespace :npm do
+  after 'npm:install', 'npm:build'
```

Comment:
what is the full deployment command now?

Reply:
same as before, cap <env> deploy.